src/cimplog-rdk-logger.c

```diff
@@ -83,3 +83,58 @@ const char *rdk_logger_module_fetch(void)
     return NULL;
 }
 
+/*
+*
+* API to log different component logs in a same log file
+*/
+
+void __cimplog_generic(const char *rdk_logger_module, const char *module, int level, const char *msg, ...)
+{
+    static int init_done = 0;
+
+    if( !init_done )
+    {
+        rdk_logger_init("/etc/debug.ini");
+        if( NULL == rdk_logger_module )
+        {
+            fprintf(stderr, "\nERROR: RDK Logger not integrated for this module !!!\n");
+            fprintf(stderr, " Provide cimplog method \"const char *rdk_logger_module_fetch(void)\" to get log prints !!\n");
```

Comment:
"const char *rdk_logger_module_fetch(void)" is not required, for this additional logger method.
Add logger error stating that "rdk_logger_module" is mandatory argument.